.ci/generate-certs.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
```

Comment:
What about https://github.com/opensearch-project/opensearch-rs/blob/833c177667ca097c7037f6991bbacfa3ee30b757/.ci/certs/generate-certs.sh?

Reply:
This PR deletes the scripts out of `.ci/certs` placing the new one directly in `.ci`
![Screenshot 2023-03-16 at 1 15 32 PM](https://user-images.githubusercontent.com/1222964/225476352-6fd61335-7175-402a-8d4f-36c460fdd95c.png)

Reply:
Ah! 🤦🏼‍♀️